.gulp/gulpfile.iced

```diff
@@ -1,49 +1,49 @@
-# Settings
-Import
-  initialized: false
-  solution: "#{basefolder}/autorest.typescript.sln"
-  sourceFolder:  "#{basefolder}/src/"
-
-# ==============================================================================
-# Tasks
-
-task 'init', "" ,(done)->
-  Fail "YOU MUST HAVE NODEJS VERSION GREATER THAN 7.10.0" if semver.lt( process.versions.node , "7.10.0" )
-  done()
-  
-# Run language-specific tests:
-task 'test', "", [], (done) ->
-  await execute "npm test", { cwd: './test/vanilla/' }, defer code, stderr, stdout
-  await execute "npm test", { cwd: './test/azure/' }, defer code, stderr, stdout
-  done();
-
-# CI job
-task 'testci', "more", [], (done) ->
-  # install latest AutoRest
-  await autorest ["--latest"], defer code, stderr, stdout
-
-  ## TEST SUITE
-  global.verbose = true
-  await run "test", defer _
-
-  ## REGRESSION TEST
-  global.verbose = false
-  # regenerate
-  await run "regenerate", defer _
-  # diff ('add' first so 'diff' includes untracked files)
-  await  execute "git add -A", defer code, stderr, stdout
-  await  execute "git diff --staged -w", defer code, stderr, stdout
-  # eval
-  echo stderr
-  echo stdout
-  throw "Potentially unnoticed regression (see diff above)! Run `npm run regenerate`, then review and commit the changes." if stdout.length + stderr.length > 0
+require './common.iced'
+
+# ==============================================================================
+# tasks required for this build 
+Tasks "dotnet"  # dotnet functions
+Tasks "regeneration"
+Tasks "publishing"
+
+# ==============================================================================
+# Settings
+Import
+  initialized: false
+  solution: "#{basefolder}/autorest.typescript.sln"
+  sourceFolder:  "#{basefolder}/src/"
+
+# ==============================================================================
+# Tasks
+
+task 'init', "" ,(done)->
+  Fail "YOU MUST HAVE NODEJS VERSION GREATER THAN 7.10.0" if semver.lt( process.versions.node , "7.10.0" )
+  done()
+  
+# Run language-specific tests:
+task 'test', "", [], (done) ->
+  await execute "npm test", { cwd: './test/vanilla/' }, defer code, stderr, stdout
+  await execute "npm test", { cwd: './test/azure/' }, defer code, stderr, stdout
+  done();
+
+# CI job
+task 'testci', "more", [], (done) ->
+  # install latest AutoRest
+  await autorest ["--latest"], defer code, stderr, stdout
+
+  ## TEST SUITE
+  global.verbose = true
+  await run "test", defer _
+
+  ## REGRESSION TEST
+  global.verbose = false
+  # regenerate
+  await run "regenerate", defer _
+  # diff ('add' first so 'diff' includes untracked files)
+  await  execute "git add -A", defer code, stderr, stdout
+  await  execute "git diff --staged -w --ignore-all-space", defer code, stderr, stdout
+  # eval
+  echo stderr
+  echo stdout
+  throw "Potentially unnoticed regression (see diff above)! Run `npm run regenerate`, then review and commit the changes." if stdout.trim().length + stderr.trim().length > 0
```

Comment:
did you see this happen (whitespace in stdout/err)? would be good to know, since we have a ton of [other repos](http://azure.github.io/autorest/trampoline.html#code-generation) with the exact same check 🙂

Reply:
no I just had a lot of different problems with the build on my machine and pushed it as an accident. I don't think that this is part of any issue. sorry to bother you! I already closed the pull request as it was messed up .... sorry.